cmd/kubeadm/app/preflight/checks.go

```diff
@@ -999,19 +999,27 @@ func RunJoinNodeChecks(execer utilsexec.Interface, cfg *kubeadmapi.NodeConfigura
 			criCtlChecker)
 	}
 
+	var bridgenf6Check Checker
 	for _, server := range cfg.DiscoveryTokenAPIServers {
 		ipstr, _, err := net.SplitHostPort(server)
 		if err == nil {
-			if ip := net.ParseIP(ipstr); ip != nil {
-				if ip.To4() == nil && ip.To16() != nil {
-					checks = append(checks,
-						FileContentCheck{Path: bridgenf6, Content: []byte{'1'}},
-					)
-					break // Ensure that check is added only once
+			checks = append(checks,
+				HTTPProxyCheck{Proto: "https", Host: ipstr},
+			)
+			if bridgenf6Check != nil {
```

Comment:
Isn't it here it should be `if bridgenf6Check == nil` ?

Reply:
My bad. Should be fixed now.